README.md

```diff
@@ -22,52 +22,35 @@ allprojects {
 }
 ```
 
-Then, add Mastodon4j dependencies to your module Gradle file:
+Then, add Bigbone dependencies to your module Gradle file:
 
 ```groovy
-implementation 'com.github.andregasser.mastodon4j:mastodon4j:$version'
-implementation 'com.github.andregasser.mastodon4j:mastodon4j-rx:$version'
+implementation 'com.github.andregasser.bigbone:bigbone:$version'
+implementation 'com.github.andregasser.bigbone:bigbone-rx:$version'
 ```
 
-Usage examples for Mastodon4j can be found in [USAGE.md](USAGE.md).
+Usage examples for Bigbone can be found in [USAGE.md](USAGE.md).
 
-## API Documentation
+## Mastodon API 
 
-The official Mastodon API can be found here: https://docs.joinmastodon.org/client/intro/
+Bigbone aims to implement the Mastodon API. The official Mastodon API can be found here: https://docs.joinmastodon.org/client/intro/
+More information about the current implementation progress can be found in [API.md](API.md). For more details about future releases 
+and our roadmap please have a look in the __Issues__ or __Projects__ section. 
 
-Implementation progress by Mastodon4j can be found in [API.md](API.md). Mastodon4j uses [Semantic Versioning 2.0.0](http://semver.org/spec/v2.0.0.html).
+## Releases
+We did not yet release a publicly available version. But we are working on it and hope to release an initial version 2.0.0 soon. Until then,
+you can experiment with our snapshot builds on JitPack.
 
+Bigbone uses [Semantic Versioning 2.0.0](http://semver.org/spec/v2.0.0.html).
 
 # Contribution
-
-## Pull Requests
-I am happy to handle any pull requests that come in. Thanks for your contribution!
-
-## Reporting Issues
-Please check out the [issues](https://github.com/andregasser/mastodon4j/issues) page for reporting issues. 
+Contributors are very welcome. If you think you can contribute, please do so! We will happily review any request we get. You can either
+create a pull request or [create an issue](https://github.com/andregasser/bigbone/issues). 
 
 # License
+Bigbone is a fork of the original library published by Toshihiro Yagi. Unfortunately, it became abandoned and has not seen any updates
+since a long time. I have therefore decided to revive and enhance it in an independent way. My personal hope is, that this library will
+satisfy developers working on the Mastodon related topics. 
 
-```
-MIT License
```

Comment:
The license is present in LICENSE.md already. That's why I have removed it here.

Reply:
Do we need to change the "copyright" line on top of LICENSE.md, or does that one just stay "Copyright (c) 2017 Toshihiro Yagi" forever?

Reply:
We could add another line on top saying something like "Copyright (c) 2022 Bigbone contributors". But we need to keep the original copyright statement in place.